testing/internal/e2e/boundary/user.go

```diff
@@ -49,9 +49,20 @@ func CreateNewUserCli(t testing.TB, ctx context.Context, scopeId string) string
 	t.Cleanup(func() {
 		AuthenticateAdminCli(t, context.Background())
 		output := e2e.RunCommand(ctx, "boundary",
-			e2e.WithArgs("users", "delete", "-id", newUserId),
+			e2e.WithArgs("users", "delete", "-id", newUserId, "-format", "json"),
 		)
-		require.NoError(t, output.Err, string(output.Stderr))
+		// Only allow an error if it's due to a Resource Not Found (404)
+		// This was needed since there's a test that already deletes the user and would run into an
+		// error during cleanup
+		if output.Err != nil {
+			var response e2e.CliError
+			err := json.Unmarshal(output.Stderr, &response)
+			require.NoError(t, err)
+			statusCode := response.Status
+			require.Condition(t, func() bool {
+				return statusCode == 404
+			})
```

Comment:
Testify is a bit particular about types when using literals, this should work
```suggestion
			require.Equal(t, int(response.Status), 404)
```